LayoutTests/svg/custom/global-constructors-expected.txt

```diff
@@ -175,7 +175,7 @@ PASS SVGAnimateElement.toString() is "function SVGAnimateElement() {\n    [nativ
 PASS SVGSetElement.toString() is "function SVGSetElement() {\n    [native code]\n}"
 PASS SVGAnimateMotionElement.toString() is "function SVGAnimateMotionElement() {\n    [native code]\n}"
 PASS SVGMPathElement.toString() is "function SVGMPathElement() {\n    [native code]\n}"
-PASS SVGAnimateColorElement.toString() is "function SVGAnimateColorElement() {\n    [native code]\n}"
+PASS SVGAnimateColorElement threw exception ReferenceError: Can't find variable: SVGAnimateColorElement.
```

Comment:
Can you change the subtest to test this is undefined or remove it?

Reply:
Can remove it. 👍

Reply:
Removed.